docs/support-bundle-guide.md

```diff
@@ -0,0 +1,203 @@
+# Support Bundle User Guide
+
+## What is Support Bundle
+
+Antrea supports collecting support bundle tarballs, which include the information
+from Antrea Controller and Antrea Agent. It is useful when troubleshooting the issues
+in Kubernetes clusters and external Nodes.
+
+**Be aware that the generated support bundle includes a lot of information,
+including logs, so please review the contents of the directory before sharing
+it on Github and ensure that you do not share anything sensitive.**
+
+There are two ways of generating support bundles. Firstly, You can run `antctlsupportbundle`
+directly in the Antrea Agent Pod, Antrea Controller Pod, or on a host with `kubeconfig`
+targeting the cluster. Secondly, You can also apply `SupportBundleCollection` CRs
+to create support bundles for Kubernetes Nodes or external Nodes. We use `SupportBundleCollection`
+to designate this feature. The details will be provided in section [Use case example](#use-case-example).
+
+## Table of Contents
+
+<!-- toc -->
+- [Prerequisites](#prerequisites)
+- [Use case example](#use-case-example)
+  - [Running antctl commands](#running-antctl-commands)
+  - [Applying SupportBundleCollection CRD](#applying-supportbundlecollection-crd)
+- [List of collected items](#list-of-collected-items)
+- [Limitations](#limitations)
+<!-- /toc -->
+
+## Prerequisites
+
+`SupportBundleCollection` is introduced in Antrea v1.10.0 as an alpha feature. The feature gate
+must be enabled in both antrea-controller and antrea-agent configurations. If you plan
+to collect support bundle on a specific external Node, you should enable it in the configuration
+on the external Node as well.
+
+```yaml
+  antrea-agent.conf: |
+    featureGates:
+    # Enable collecting support bundle files with SupportBundleCollection CRD.
+    SupportBundleCollection: true
+```
+
+```yaml
+  antrea-controller.conf: |
+    featureGates:
+    # Enable collecting support bundle files with SupportBundleCollection CRD.
+    SupportBundleCollection: true
+```
+
+## Use case example
```

Comment:
Is it possible to introduce a section like "The SupportBundleCollection resource" to introduce the CRD before "Use case example"? Then we could guide users how to configure the fields in a CR, like the meaning and limitations of fields.

Reply:
Added. Personally I don't think we need to do the rewriting to explain the fields in the CRD one by one. Most of them are self-explaintory. Two examples in the `use case example` section are enough to illustrate all its functionalities.

Reply:
Better to add a link to refer to the supportbundlecollection yaml file to let the users know how to create the resource